tests/cpu/Config_tests.cpp

```diff
@@ -181,6 +181,36 @@ OCIO_ADD_TEST(Config, simple_config)
     OCIO_CHECK_NO_THROW(config->sanityCheck());
 }
 
+OCIO_ADD_TEST(Config, simple_config_with_duplicates)
+{
+
+    constexpr char SIMPLE_PROFILE[] =
+        "ocio_profile_version: 2\n"
+        "search_path: \n"
```

Comment:
I missed than one, please add `luts` as the directory where to find the LUT files. 
The line must be:  `"search_path: luts\n"`.